src/cli/send/junitUtils.spec.ts

```diff
@@ -0,0 +1,216 @@
+    });
+    expect(result).toBe(
+      `
+<testsuites errors="0" disabled="0" failues="0">
+  <testsuite name="test.http" tests="2" failures="0" skipped="0" package="." time="2.003" file="test.http">
+    <properties>
+      <property name="name" value="test"/>
+    </properties>
+    <testcase name="test" classname="test.http" assertions="1" time="1.001"/>
+    <properties>
+      <property name="name" value="test2"/>
+    </properties>
+    <testcase name="test2" classname="test.http" assertions="1" time="1.002"/>
+  </testsuite>
+  <testsuite name="test2.http" tests="1" failures="0" skipped="0" package="." time="1.002" file="test2.http">
+    <properties>
+      <property name="name" value="other"/>
+    </properties>
+    <testcase name="other" classname="test2.http" assertions="1" time="1.002"/>
+  </testsuite>
+</testsuites>
+    `.trim()
+    );
+  });
+
+  it('should output disabled request', () => {
+    const result = transformToJunit({
+      _meta: {
+        version: 'test',
+      },
+      requests: [
+        {
+          fileName: 'test.http',
+          disabled: true,
+          name: 'test',
+          summary: {
+            totalTests: 0,
+            successTests: 0,
+            failedTests: 0,
+          },
+        },
+      ],
+      summary: {
+        totalRequests: 1,
+        successRequests: 0,
+        disabledRequests: 1,
+        failedRequests: 0,
+        totalTests: 1,
+        successTests: 1,
+        failedTests: 0,
+      },
+    });
+    expect(result).toBe(
+      `
+<testsuites errors="0" disabled="1" failues="0">
+  <testsuite name="test.http" tests="1" failures="0" skipped="1" package="." time="0.000" file="test.http">
+    <properties>
+      <property name="name" value="test"/>
+    </properties>
+    <testcase name="test" classname="test.http" assertions="0" time="0.000">
+      <skipped/>
+    </testcase>
+  </testsuite>
+</testsuites>
+    `.trim()
+    );
+  });
+
+  it('should output invalid request', () => {
+    const result = transformToJunit({
+      _meta: {
+        version: 'test',
+      },
+      requests: [
+        {
+          fileName: 'test.http',
+          disabled: false,
+          name: 'test',
+          summary: {
+            totalTests: 2,
+            successTests: 1,
+            failedTests: 1,
+          },
+          testResults: [
+            {
+              result: false,
+              message: 'Assertions fail',
+              error: {
+                displayMessage: 'failed result',
+                error: { message: 'test', name: 'unknown', stack: '' } as unknown as Error,
+              },
+            },
+          ],
+        },
+      ],
+      summary: {
+        totalRequests: 1,
+        successRequests: 0,
+        disabledRequests: 0,
+        failedRequests: 1,
+        totalTests: 2,
+        successTests: 1,
+        failedTests: 1,
+      },
+    });
+    expect(result).toBe(
+      `
+<testsuites errors="0" disabled="0" failues="1">
```

Comment:
when we write to stdout will the xml frontmatter be included or do we need to do something to add:

`<?xml version="1.0" encoding="UTF-8"?>`

Reply:
xml formatter added using `createProcessingInstruction`